templates/tfengine/components/audit/main.tf

```diff
@@ -40,84 +40,67 @@ resource "google_{{.parent_type}}_iam_audit_config" "config" {
   }
 }
 
-# BigQuery log sink.
-resource "google_logging_{{.parent_type}}_sink" "bigquery_audit_logs_sink" {
-  name                 = "bigquery-audit-logs-sink"
-  {{$parent_field}}    = {{$parent_var}}
-  include_children     = true
-  filter               = "{{$filter}}"
-  destination          = "bigquery.googleapis.com/projects/${module.project.project_id}/datasets/${module.bigquery_destination.bigquery_dataset.dataset_id}"
+module "bigquery_export" {
+  source  = "terraform-google-modules/log-export/google"
+  version = "~> 5.0.0"
+
+  log_sink_name          = "bigquery-audit-logs-sink"
+  destination_uri        = "${module.bigquery_destination.destination_uri}"
+  filter                 = "{{$filter}}"
+  parent_resource_type   = "{{.parent_type}}"
+  parent_resource_id     = {{$parent_var}}
+  unique_writer_identity = true
+  include_children       = true
 }
 
 module "bigquery_destination" {
-  source  = "terraform-google-modules/bigquery/google"
-  version = "~> 4.3.0"
-
-  dataset_id                  = "{{.logs_bigquery_dataset.dataset_id}}"
-  project_id                  = module.project.project_id
-  location                    = "{{.bigquery_location}}"
-  default_table_expiration_ms = 365 * 8.64 * pow(10, 7) # 365 days
-  access = [
-    {
-      role          = "roles/bigquery.dataOwner",
-      special_group = "projectOwners"
-    },
```

Comment:
do we still grant this permissions separately?

Reply:
dataViewer is done by the project iam permission below (made it bigquery.user)

Converting BQ permissions away from default is being tracked by https://github.com/terraform-google-modules/terraform-google-log-export/issues/49